addons/website/static/src/js/utils.js

```diff
@@ -375,6 +375,24 @@ function generateGMapLink(dataset) {
             + '&ie=UTF8&iwloc=&output=embed';
 }
 
+/**
+ * Returns the parsed data coming from the data-for element for the given form.
+ *
+ * @param {string} formId
+ * @returns {Object|undefined} the parsed data
+ */
+function getParsedDataFor(formId) {
+    const dataForEl = document.querySelector(`[data-for='${formId}']`);
+    if (!dataForEl) {
+        return;
+    }
+    return JSON.parse(dataForEl.dataset.values
+        .replace('False', '""')
+        .replace('None', '""')
+        .replace(/'/g, '"')
```

Comment:
I leave the wrong code here and the fw-port of [this other PR](https://github.com/odoo/odoo/pull/124975) will fix it :+1: 

and this is ready for your review @qsm-odoo

Reply:
That other PR has been merged and forward-ported, I guess this has to be adapted, no? :+1: